moveit_core/robot_state/test/test_kinematic.cpp

```diff
@@ -33,14 +33,19 @@
 *********************************************************************/
```

Comment:
rename file to ``robot_state_test.cpp``

"kinematic" is the old name of robot_state long long ago